examples/ex1p.cpp

```diff
@@ -65,11 +65,15 @@ int main(int argc, char *argv[])
 
    // 2. Parse command-line options.
    const char *mesh_file = "../data/star.mesh";
+   //const char *mesh_file = "../data/beam-hex.mesh";
+
    int order = 1;
    bool static_cond = false;
    bool pa = false;
+   bool amgx = true;
    const char *device_config = "cpu";
```

Comment:
During testing, I left the MFEM device set to CPU. I think AmgX has a way to identify if the memory is the host or device and do the transfers accordingly. Something, I'd like to inquire about.

Reply:
I'm interested in this too! Based on just a few runs it seems to produce identical output with device `cuda` and `cpu`, I'm curious what it's actually doing.